### ISSUE TITLE
HTTPS Certification of nonius.io is expired

### ISSUE THREAD

**Issue by michalfita** [libnonius/nonius#106]
I'd like to use nonius library, but all the documentation is available under [nonius.io](https://nonius.io) website using HSTS, hence I can't open that website as its certificate **expired** on 5 October 2018.

Could you please fix that?

**Comment by Donaldpherman** [comment]
Is this project still being supported?

**Comment by michalfita** [comment]
I don't think so... last time I checked the domain `nonius.io` expired and has been taken over by someone.

**Comment by dotMorten** [comment]
Could we perhaps update the readme to not point to `nonius.io` ? I'm getting redirected to various ad sites, and sometimes those are flagged malicious.

**Comment by michalfita** [comment]
@rmartinho seem to lost interest in this project completely and the **libnonius** organisation has zero members. So asking anyone about anything probably doesn't make any sense any more.
